### PR TITLE
Plant skill unlock

### PR DESCRIPTION
**Pull request by cedric-h**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -57,6 +57,6 @@ futures-channel = "0.3.5"
 
 [dependencies.hcor]
 git = "https://github.com/hackagotchi/hcor.git"
-branch = "slim"
+branch = "plant_skill_unlock"
```
Comment (Muirrum) [inline Cargo.toml]: shouldn't this target staging/master?

File: src/wormhole/session/tile/plant/mod.rs [hunk 2]
```diff
@@ -43,15 +60,15 @@ async fn plant_craft() -> hcor::ClientResult<()> {
     // (it probably fails because it's already been established in another test)
     drop(pretty_env_logger::try_init());
 
-    let seed_arch = hcor::CONFIG
-        .possession_archetypes
+    let seed_config = hcor::CONFIG
+        .possession_configetypes
```
Comment (Muirrum) [inline src/wormhole/session/tile/plant/mod.rs:64]: these lines are commented out, should they be?

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (cedric-h) [reply Cargo.toml]: yes indeed, good catch

File: src/wormhole/session/tile/plant/mod.rs (hunk 2 above)
Comment (cedric-h) [reply src/wormhole/session/tile/plant/mod.rs:64]: meh, it's just some test boilerplate for an unimplemented route. next PR uncomments it and puts it to work.

**Pull request closed without merging**